src/wire/mod.rs

```diff
@@ -153,6 +155,11 @@ pub use self::icmpv4::{Message as Icmpv4Message,
                        Packet as Icmpv4Packet,
                        Repr as Icmpv4Repr};
 
+#[cfg(feature = "proto-ipv4")]
+pub use self::igmpv2::{Packet as IgmpPacket,
```

Comment:
nit: seems strange to me to have the module named `igmpv2` but the exported structures and enums named `Igmp<Something>`. Would it make sense to just leave the module named `igmp` since we're not going to allow `Igmpv1Packet`, `Igmpv2Packet`, etc?

Reply:
I completely agree. Renamed to `igmp`!